<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordToDictionaryForm</class>
 <widget class="QMainWindow" name="WordToDictionaryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>zkanji - Add word to dictionary</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Select one or more meanings to copy:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="switchButton">
            <property name="toolTip">
             <string>Switch to adding word to group</string>
            </property>
            <property name="text">
             <string>Word to group...</string>
            </property>
            <property name="icon">
             <iconset resource="Resources/resources.qrc">
              <normaloff>:/togroup.svg</normaloff>:/togroup.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="wordsWidget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="ZDictionaryListView" name="wordsTable">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>1</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ZStatusBar" name="wordsStatus"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="destLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Destination:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="dictCBox"/>
          </item>
          <item>
           <widget class="QDialogButtonBox" name="buttonBox">
            <property name="standardButtons">
             <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="meaningsWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Existing words in the destination dictionary:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="ZDictionaryListView" name="meaningsTable"/>
           </item>
           <item>
            <widget class="ZStatusBar" name="meaningsStatus"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZStatusBar</class>
   <extends>QStatusBar</extends>
   <header>zstatusbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ZDictionaryListView</class>
   <extends>QTableView</extends>
   <header>zdictionarylistview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
